front/app/api/project_files/useAddProjectFile.ts

```diff
@@ -17,16 +16,8 @@ const addProjectFile = async ({
   });
 
 const useAddProjectFile = () => {
-  const queryClient = useQueryClient();
   return useMutation<IProjectFile, CLErrors, AddProjectFileObject>({
     mutationFn: addProjectFile,
-    onSuccess: (_data, variables) => {
-      queryClient.invalidateQueries({
-        queryKey: projectFilesKeys.list({
-          projectId: variables.projectId,
-        }),
```

Comment:
Note: I removed the cache invalidation on these individual hooks for the Project Files, since we were doing it too many times. Now I only invalidate once in the `useSyncProjectFiles` hook :)